### PR TITLE
VJ/styled login page

### PR DESCRIPTION
**Pull request by vivekjain1397**
Styled login page (title, tagline, google sign in button, gradient ("#90BEDE", -> "#C7F9FF")

### CODE REVIEW

File: imdown-mobile/App.tsx [hunk 1]
```diff
@@ -7,30 +7,37 @@ import { createStackNavigator } from "@react-navigation/stack";
...
+import { Ubuntu_400Regular, Ubuntu_400Regular_Italic, Ubuntu_700Bold, useFonts } from '@expo-google-fonts/ubuntu';
 import { Inter_400Regular } from '@expo-google-fonts/inter';
+import { AppLoading } from 'expo';
 
 
 const Stack = createStackNavigator();
 
 export default function App() {
-  useFonts({
+  let [fontsLoaded] = useFonts({
```
Comment (abhishyantkhare) [inline imdown-mobile/App.tsx:18]: I think this can be `const` not `let`

Comment (abhishyantkhare) [reply imdown-mobile/App.tsx:18]: but it doesn't really matter lmao

**Comment by brianlevis** [comment]
> Looks dope but I think your git history is fucked up LOL you have a bunch of commits thats not your commits. I think you have to properly rebase against master

Maybe the issue is your local master is out of date - try running
```
git pull --rebase origin master
```

**Comment by brianlevis** [comment]
```
git rebase --onto origin/master 041f579db4f58b2c770f2898127e6430fd2b6597 vivek-styling-login-page
```
run dis ^

**Pull request merged**